tests/cross_functional/performance/csi_tests/test_pvc_multi_snapshot_performance.py

```diff
@@ -176,7 +177,13 @@ def teardown(self):
 
             # Deleting the Data pool
             log.info(f"Deleting the test storage pool : {self.sc_name}")
-            self.delete_ceph_pool(self.sc_name)
+            try:
+                self.delete_ceph_pool(self.sc_name)
+            except CommandFailed as cmdFailedEx:
+                log.info(f"Exception caught {cmdFailedEx}")
+                log.info("Continue to teardown ")
+                if "pas-testing-rbd" in cmdFailedEx and "not found" in cmdFailedEx:
```

Comment:
Does this validation required?

Reply:
@ramkiperiy  This is an acceptable exception that should be ignored by the test when appears, and the explicit validation was added following Karthicks request.